### PR TITLE
Plugins readme + Hubspot, Zoho, vTiger, Salesforce CRM and Mailchimp docs created

### PR DESCRIPTION
**Pull request by escopecz**
Merge this PR after the Mautic 1.2.3 is released.

### CODE REVIEW

File: plugins/mailchimp.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Mailchimp integration
+
+Mautic can send leads to Mailchimp upon some lead's action or when it gain some point limit. Mailchimp can be also used as an email sending system. This option is not part of the plugin though so it will not be covered on this page.
```
Comment (alanhartless) [inline plugins/mailchimp.md]: `Mailchimp can be also used as an email sending system` may confuse some as it's not Mailchimp but Mandrill which is a service of Mailchimp.  I'm not sure I would mention that at all as it's not related to the Mailchimp integration.

File: plugins/zoho_crm.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Mautic - Zoho CRM plugin
+
+This plugin can push a lead to Zoho CRM when a lead makes some action.
+
+If you don't have the Zoho CRM account yet, [create it](https://www.zoho.com/crm/).
+
+## Configure the Zoho CRM plugin
+
+Insert the email and password you created the Zoho account with into the Mautic Zoho integration plugin and authorize it. Set the *Publish* switch to *Yes*. Save.
```
Comment (alanhartless) [inline plugins/zoho_crm.md]: Might mention here that if Zoho Two Factor Authentication is enabled, an Application Specific Password will need to be generated and used. https://www.zoho.com/mail/help/adminconsole/two-factor-authentication.html#alink5

File: plugins/mailchimp.md (hunk 1 above)
Comment (escopecz) [inline plugins/mailchimp.md]: Agreed

File: plugins/zoho_crm.md (hunk 2 above)
Comment (escopecz) [inline plugins/zoho_crm.md]: added

**Comment by alanhartless** [comment]
We'll merge this once we get the latest changes released.

**Pull request merged**